### ISSUE TITLE
The BraTS21 result

### ISSUE THREAD

**Issue by shanwq** [aigzhusmart/Slim-UNETR#2]
Hi, your work is wonderful!
I'd like to ask about how you got the BraTS21 results in your paper.
You said you are using the whole training set of 1251 cases for your model training. So your results of avg-dice 92.44% are from the 219 validation set? Since their labels are not available, did you submit your prediction results of the 219 cases to the official Synapse website? Can I see your results on the Synapse Validation Phase Snapshot website? (https://www.synapse.org/#!Synapse:syn25829067/wiki/612712)
Thanks a lot!

**Comment by DTDWJ365** [comment]
I believe this is a point that deserves some thought.

**Comment by hhuxuchang** [comment]
I also have this problem.
